content/installation/net/install/Supported-technology-net.md

```diff
@@ -20,6 +20,7 @@ The Contrast .NET agent supports analysis of web applications built on the follo
 >**Notes:**
 > * The **Mono** runtime isn't supported because Mono doesn't have a **Profiler API**. The Profiler API is an interface based on a **Component Object Model (COM)**, and isn't supported on Linux.
 > * The **Classic ASP** language isn't supported because Classic ASP applications don't run on the .NET runtime.
+> * The .NET Framework agent does NOT support .NET framework applications running within the ASP.NET Core pipeline. The agent also doesn't support ASP.NET Core applications running under the .NET framework.
```

Comment:
This confuses me slightly. Could we simply state:
> The .NET Framework agent does NOT support analysing .NET Framework ASP.NET Core applications.

But I'm not heavily opinionated either way.

Reply:
Good suggestion. Adopted.